.github/workflows/ci.yml

```diff
@@ -217,6 +217,7 @@ jobs:
       # SOS
       - name: Build SOS (${{ matrix.sos_config }})
         run: |
+          git submodule update --remote
```

Comment:
Sounds good - we can keep them separate. Figured I might as well test it out with this PR and then revert the CI changes after finding out what works, then push the CI changes in a different PR. Even though I don't expect any issues with the ibput/ibget tests with different test configurations, it would be great to validate this PR with the new tests incorporated in the CI